wqflask/wqflask/views.py

```diff
@@ -376,6 +377,12 @@ def wcgna_setup():
     return render_template("wgcna_setup.html", **request.form)
 
 
+@app.route("/wgcna_results", methods=('POST',))
```

Comment:
**suggestion:** Next time when creating routes, have a look at: https://restfulapi.net/resource-naming/

My goal with this project is to eventually have routes and APIs that are easy to maintain. Small improvements will have compounding effects. I'll refer you to a PR I'll submit later today on the auth work. Cc @zsloan